examples/protobuf/GenericHost/README.md

```diff
@@ -0,0 +1,23 @@
+# GenericHost
+
+This example shows how to use dependency injection and the .NET Generic Host with IceRPC client and server applications.
+
+You can build the client and server applications with:
+
+``` shell
+dotnet build
+```
+
+First start the Server program:
+
+```shell
+cd Server
+dotnet run
+```
+
+In a separate window, start the Client program:
```

Comment:
Should this be "In a separate terminal"?

Reply:
Or seperate shell, separate tab would also work.

Reply:
I will fix in a separate PR, need to fix all examples